tests/test_manager_page.py

```diff
@@ -0,0 +1,136 @@
+        - В списке клиентов появился новый клиент с заданными именем и фамилией     
+    """
+)
+@pytest.mark.parametrize("surname, post_code",
+                         [
+                             ("rand_str", "rand_num")
+                         ])
+def test_add_customer_successfully(request, browser, surname, post_code):
+    post_code_result = request.getfixturevalue(post_code)
+    surname_result = request.getfixturevalue(surname)
+    with allure.step("Открытие страницы https://www.globalsqa.com/angularJs-protractor/BankingProject/#/manager"):
+        page = ManagerPage(browser)
+        page.open()
+
+    with allure.step("Открытие формы создания клиента"):
+        page.open_add_customers_form()
+
+    with allure.step("Заполнение формы создания нового клиента"):
+        name = create_word_from_number(post_code_result)
+        page.fill_customers_form(name, surname_result, post_code_result)
+        page.click_submit_form_button()
+
+    with allure.step("Проверка появления всплывающего уведомления"):
+        expected_alert_text = "Customer added successfully with customer id :"
+        actual_alert_text = page.get_text_from_alert_and_close()
+        assert expected_alert_text in actual_alert_text, "Текст об успешном создании клиента не найден"
+
+    with allure.step("Проверка добавления созданного клиента в таблицу"):
+        page.open_customers_data()
+        customer_list = page.get_text_from_table()
+        assert name and surname_result in customer_list
+
+
+@allure.title("Сортировка списка клиентов по имени")
+@allure.description(
+    """
+    Цель: Проверить сортировку списка клиентов по имени
+    
+    Предусловие:
+        - Открыть браузер
+    
+    Шаги:
+        1. Открыть страницу "https://www.globalsqa.com/angularJs-protractor/BankingProject/#/manager"  
+        2. Нажать кнопку Customers
+        3. Кликнуть один раз на заголовок столбца First Name
+        4. Проверить сортировку имен в алфавитном порядке по убыванию
+        
+    Ожидаемый результат:
+        - Клиенты отсортировались по имени     
+    """
+)
+def test_alphabet_order(browser):
+    with allure.step("Открытие страницы https://www.globalsqa.com/angularJs-protractor/BankingProject/#/manager"):
+        page = ManagerPage(browser)
+        page.open()
+
+    with allure.step("Открытие списка клиентов"):
+        page.open_customers_data()
+
+    with allure.step("Получение несортированных данных из таблицы"):
+        unsorted_table = convert_table_data_to_list(page.get_text_from_table())
+
+    with allure.step("Сортировка клиентов по имени"):
+        page.sort_by_first_name()
+
+    with allure.step("Проверка сортировки списка клиентов по имени"):
+        expected_sorted_table = sorted(unsorted_table, reverse=True, key=lambda i: i[0])
+        actual_sorted_table = convert_table_data_to_list(page.get_text_from_table())
+        assert expected_sorted_table == actual_sorted_table, "Данные не отсортировались по столбцу first name"
```

Comment:
Здесь и далее. Можно сократить код.
```suggestion
    with allure.step("Проверка сортировки списка клиентов по имени"):
        assert sorted(unsorted_table, reverse=True, key=lambda i: i[0])== convert_table_data_to_list(page.get_text_from_table()), "Данные не отсортировались по столбцу first name"
```

Reply:
Мне кажется, что таким сокращением мы теряем читаемость кода. Получется очень громоздкий однострочник, где сразу и не поймешь что с чем сравнивается. А говорящие названия переменных повышают читаемость кода.